hw-js/js/app.js

```diff
@@ -0,0 +1,79 @@
+
+// interpolation- Using a variable to represent a value.
+
+var num = 20
+
+console.log(5 + num);
+
+// concatenation- to combine; put 2 strings together
+
+const pet = "dog";
+const petName = "Eddie";
+
+console.log("I " + " have a " + pet + " and his name is " + petName + "." );
+
+
+// DRY- Don't repeat yourself. We use loops for this, and is an easier way to to repeat.
+
+//BOOLEAN
+
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b); 
+console.log(c > d); 
+console.log('Name' === 'Name');
+console.log(a < b && b < c); 
+console.log(a === a && a < d); 
+console.log( e === e);
+console.log( 48 == '48');
+
+
+//section 3- This is an infinite loop, since while is true, it will infinitly console.log
+//section 3-2- This is not an infinite loop, RunProgram var is true, but the console.log is false.
```

Comment:
you're correct that it's not an infinite loop, but your stated reasoning isn't quite right:
* runProgram should *become* false during the loop
* but actually what happens is you get an error because you're trying to change the value of a const 
* Also a console.log cannot be false -- it doesn't technically have a boolean value